test/framework/shoot_utils.go

```diff
@@ -83,6 +90,38 @@ func (f *ShootFramework) GetValiLogs(ctx context.Context, valiLabels map[string]
 	return search, nil
 }
 
+func isValiAuthEnabled(ctx context.Context, log logr.Logger, selector labels.Selector, namespace string, c kubernetes.Interface) bool {
+	//Check if the auth is enabled in the configuration of the target
+	enabled, ok := authEnabledMap[selector.String()]
+	if ok {
+		return enabled
+	}
+
+	commandAuthEnabled := fmt.Sprintf("wget 'http://localhost:%d/config' -O-", valiPort)
+	r, err := PodExecByLabel(ctx, selector, valiLogging, commandAuthEnabled, namespace, c)
+	if err != nil {
+		log.Error(err, "Failed to check the vali config endpoint")
+		return false
+	}
+	scanner := bufio.NewScanner(r)
+	for scanner.Scan() {
+		line := scanner.Text()
```

Comment:
Must it be so "complicated" in the context of this test code, or would `strings.Contains(r, "auth_enabled: true`) be good enough?

Reply:
sure, proposal works as well